static/js/webworker-around.js

```diff
@@ -0,0 +1,16 @@
+onmessage = _ => {
+
+  fetch('/around/spots.json')
+    .then(response => response.json())
+    .then(items => {
+      items.forEach(item => {
+        let name = item.name
+        let wifi = item.wifiQuality
+        let power = item.powerAvailable
+        let comment = item.comments[0]
+        let lng = item.position.longitude
+        let lat = item.position.latitude
+        postMessage([name,wifi,power,comment,lng,lat])
```

Comment:
oups I forget ... I change that right now and I'll do a `git push -f`and a merge :) youhouuuu ^^